test/e2e/autoscaling/horizontal_pod_autoscaling.go

```diff
@@ -104,20 +108,69 @@ var _ = SIGDescribe("[Feature:HPA] Horizontal pod autoscaling (scale resource: C
 			doNotScaleOnBusySidecar("rs", e2eautoscaling.KindReplicaSet, true, f)
 		})
 	})
+
+	ginkgo.Describe("CustomResourceDefinition", func() {
+		ginkgo.It("Should scale with a CRD targetRef", func() {
+			scaleTest := &HPAScaleTest{
+				initPods:                    1,
+				totalInitialCPUUsage:        150,
+				perPodCPURequest:            200,
+				targetCPUUtilizationPercent: 50,
+				minPods:                     1,
+				maxPods:                     2,
+				firstScale:                  2,
+				targetRef:                   e2eautoscaling.CustomCRDTargetRef(),
+			}
+			scaleTest.run("rc-light", e2eautoscaling.KindRC, f)
```

Comment:
crd-light? This is a name for resource consumer, it was rc-light in the other test because it was about ReplicationController.

Reply:
Thanks for noticing that, resolved.